tests/v2/validation/charts/monitoring.go

```diff
@@ -266,6 +266,7 @@ func createPrometheusRule(client *rancher.Client, clusterID string) error {
 	if err != nil {
 		return err
 	}
+	noDuration := monitoringv1.Duration("0s")
```

Comment:
why are we updating this ?

Reply:
it is broken right now, according to my editor